.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,88 @@
+name: "Docker build"
+
+on:
+  workflow_call:
+    inputs:
+      sha:
+        required: true
+        type: string
+      version:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+  packages: write
```

Comment:
Do we need this here? As we're only building here and not writing packages

Reply:
Oh, you're right. 
Totally useless. 
Fixed in f2a65045